### PR TITLE
split up configuration into methods

### PR DESCRIPTION
**Pull request by dbu**
i want to improve on #245 but notice that the configuration being one huge spaghetti is starting to get into the way. this PR changes no functionality at all, just splits the configuration tree building into separate sections. can we do this? then i will rebase #245 and try to push validation into the configuration definition.

### CODE REVIEW

File: DependencyInjection/Configuration.php [hunk 1]
```diff
@@ -49,105 +50,149 @@ public function getConfigTreeBuilder()
...
+        $this->loadPersistence($nodeBuilder);
+        $this->loadSonataAdmin($nodeBuilder);
+        $this->loadAlternateLocale($nodeBuilder);
+        $this->loadErrorHandler($nodeBuilder);
+        $this->loadSitemap($nodeBuilder);
+        $this->loadContentListener($nodeBuilder);
+
+        return $treeBuilder;
+    }
+
```
Comment (ElectricMaxxx) [inline DependencyInjection/Configuration.php]: Do we need docblock for the sub-methods?

**Comment by ElectricMaxxx** [comment]
@dbu looks cool. Except my question about the docblocks, we can merge it.

**Comment by dbu** [comment]
renamed the methods to configure... and added docblocks.

**Comment by WouterJ** [comment]
The convention in Symfony core is to name these methods "addXxxSection"

**Comment by dbu** [comment]
oh, ok. makes sense. i was looking at the doctrine bundle as inspiration. but addXxSection sounds good, will change to that

**Comment by dbu** [comment]
and pushed again with the better names

**Comment by ElectricMaxxx** [comment]
its awesome how big that configuration grews up.

**Comment by ElectricMaxxx** [comment]
@dbu i just restarted the job, cause i don't understand the reason. The test failed just on php 5.6

**Comment by ElectricMaxxx** [comment]
whaat was the change?

**Comment by ElectricMaxxx** [comment]
thanks @dbu

**Comment by dbu** [comment]
no change. just restarted once more. no idea how that happened.

**Pull request merged**